importer/EsEs.py

```diff
@@ -1,38 +1,29 @@
 from Monument import Monument, Dataset
 import importer_utils as utils
 import importer as importer
+import dateparser
 
 
 class EsEs(Monument):
 
     def set_heritage_with_date(self):
+        """
+        Set heritage status (bien interes cultural).
+
+        Optionally, with start date qualifier.
+        """
         heritage = self.mapping["heritage"]["item"]
         if self.has_non_empty_attribute("fecha"):
             # 20 de febrero de 1985
-            months = ["enero", "febrero", "marzo",
-                      "abril", "mayo", "junio",
-                      "julio", "agosto",
-                      "septiembre", "octubre",
-                      "noviembre", "diciembre", ]
-            date_parts = self.fecha.split(" ")
-            if len(date_parts) != 5:
+            es_date = dateparser.parse(self.fecha, languages=['es'])
+            try:
+                date_dict = utils.datetime_object_to_dict(es_date)
+            except ValueError:
```

Comment:
add `, start_time` as the third argument to "add_to_report" to tie it to a property?

Reply:
Added!